### PR TITLE
🚀 add deploy script tbtc and rdnt

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

**Review by zekiblue — APPROVED** [review]
lgtm

File: script/utils/Constants.sol [hunk 1]
```diff
@@ -104,6 +108,8 @@ library OracleIds {
     uint80 internal constant ETH_USD = 2;
     uint80 internal constant DEGEN_USD = 9;
     uint80 internal constant EZETH_ETH = 10;
+    uint80 internal constant RDNT_USD = 13;
+    uint80 internal constant TBTC_USD = 14;
```
Comment (j-vp) [inline script/utils/Constants.sol:112]: pls put them below the 11 & 12

Comment (Thomas-Smets) [reply script/utils/Constants.sol:112]: it's alfabetic, like all the other constants

Comment (j-vp) [reply script/utils/Constants.sol:112]: image

Comment (Thomas-Smets) [reply script/utils/Constants.sol:112]: I'm not good with the alfabet

**Review by j-vp — APPROVED** [review]
gtg

**Pull request merged**